business-registry-dashboard/app/enums/authorized-actions.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * List of actions authorized via roles. See Authorizations module.
+ * Ultimately, this will be combined into the Authorization Roles enum.
+ */
+export enum AuthorizedActions {
+  ADD_BUSINESS_NO_AUTHENTICATION,
+  ADD_NAME_REQUEST_NO_AUTHENTICATION,
```

Comment:
Since these two have the same permissions across roles, I'm thinking about combining them. What do you think about that?